Documentation/In-depth/ComposerJson/Index.rst

```diff
@@ -0,0 +1,288 @@
+.. include:: /Includes.txt
+
+
+.. _project_composer.json:
+
+=============
+composer.json
+=============
+
+In Composer, everything is a package, but there are differences between
+a "project" and a "library" and the best practices that are used within
+the file :file:`composer.json` for a project or library.
+
+This page explains the basics for your **project** (TYPO3 installation)
+:file:`composer.json`, while :file:`composer.json` for libraries (or more
+specifically for TYPO3 extensions) is handled in TYPO3 Explained
+:ref:`t3coreapi:composer-json`
+
+In a Composer based TYPO3 installation, we typically have a file structure
+like this:
+
+
+.. code-block:: none
+
+   project-root/
+   ├── composer.json
+   ├── composer.lock
+   └── public
+       ├── fileadmin
+       ├── typo3
+       ├── typo3conf
+       └── ...
+
+You may have used the `typo3/cms-base-distribution` package (as explained in
+:ref:`install-via-composer`) which created a :file:`composer.json`
+for you or you may want to create a :file:`composer.json` for your project from
+scratch.
+
+.. _project_composer.json-example:
+
+Example composer.json
+=====================
+
+.. code-block:: json
+
+   {
+      "name": "vendor/mysite",
```

Comment:
There is `type:project` which should make it clear, but I agree the name is misleading. Will fix.

Personally, I find the name "distribution" misleading. We have [different meanings for distribution](https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues/1384): e.g. as in typo3-base-distribution or in EXT:introduction. 

Not sure what to use, I could use distribution because it is the de-facto standard for TYPO3, but I personally hate it for the named reason. Would be good to get feedback on https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues/1384